packages/marko-web/components/load-more/index.marko

```diff
@@ -14,15 +15,20 @@ $ const queryParams = {
 <if(Component)>
   <marko-web-query|{ nodes, pageInfo }| name=input.queryName params=queryParams>
     <if(nodes.length)>
-      <marko-web-page-container tag="section" ...input.pageInput modifiers=["load-more", ...getAsArray(input, "page.modifiers")]>
+      <marko-web-page-container
+        tag="section"
+        ...input.pageInput
+        modifiers=["load-more", ...getAsArray(input, "page.modifiers")]
+        class=containerClass
```

Comment:
Remove `class=` and instead add the modifier... `modifiers=["load-more", `page-${pageNumber}`, ...getAsArray(input, "page.modifiers")]`